docs/csharp/distinguish-delegates-events.md

```diff
@@ -1,93 +1,43 @@
 ---
 title: Delegates vs. events
 description: Learn the difference between delegates and events and when to use each of these features of .NET Core.
-ms.date: 06/20/2016
+ms.date: 03/11/2025
 ms.subservice: fundamentals
-ms.assetid: 0fdc8629-2fdb-4a7c-a433-5b9d04eaf911
 ---
-
 # Distinguishing Delegates and Events
 
 [Previous](modern-events.md)
 
-Developers that are new to the .NET Core platform often struggle
-when deciding between a design based on `delegates` and a design
-based on `events`. The choice of delegates or events is often difficult, because the two
-language features are similar. Events are even built using
-the language support for delegates.
+Developers that are new to the .NET Core platform often struggle when deciding between a design based on `delegates` and a design based on `events`. The choice of delegates or events is often difficult, because the two language features are similar. Events are even built using the language support for delegates.
 
-They both offer a late binding scenario: they enable scenarios
-where a component communicates by calling a method that is only
-known at run time. They both support single and multiple subscriber
-methods. You may find this referred to as singlecast and multicast
-support. They both support similar syntax for adding and removing
-handlers. Finally, raising an event and calling a delegate use exactly the same method call syntax. They even both support the same `Invoke()`
-method syntax for use with the `?.` operator.
+They both offer a late binding scenario: they enable scenarios where a component communicates by calling a method that is only known at run time. They both support single and multiple subscriber methods. You might find these terms referred to as single cast and multicast support. They both support similar syntax for adding and removing handlers. Finally, raising an event and calling a delegate use exactly the same method call syntax. They even both support the same `Invoke()` method syntax for use with the `?.` operator.
```

Comment:
Some of this is still a bit confusing. I always thought that the relationship between events and delegates, was that an event handler is a delegate type.